test/db/formats/elf/main

```diff
@@ -4,6 +4,16 @@ CMDS=?v main
 EXPECT=<<EOF
 0x401126
 EOF
+EXPECT_ERR=<<EOF
```

Comment:
can we pls merge @XVilka pr to get rid of all those warnings?

Reply:
Yes @xvilka please remove those variables or use them somehow in your pr. Whatever is easier faster for you works for me.